### PR TITLE
Glusterfs: Remove unwanted `log-file` mount argument.

### PR DESCRIPTION
**Pull request by humblec**
Fixes # https://github.com/kubernetes/kubernetes/issues/68772

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note

```

### CODE REVIEW

File: pkg/volume/glusterfs/glusterfs.go [hunk 1]
```diff
@@ -319,10 +319,11 @@ func (b *glusterfsMounter) setUpAtInternal(dir string) error {
 		// its own log based on PV + Pod
 		log = path.Join(p, b.pod.Name+"-glusterfs.log")
 
+		// Use derived/provided log file in gluster fuse mount
```
Comment (wongma7) [inline pkg/volume/glusterfs/glusterfs.go]: should remove the word "provided" since it's only derived now

Comment (humblec) [reply pkg/volume/glusterfs/glusterfs.go]: Done.. thanks!

**Comment by saad-ali** [comment]
Please create a issue and link to it from this PR "Fixes #...". Once that is done I will add it to 1.12 milestone since it is as a regression.

**Comment by humblec** [comment]
@saad-ali Done.. thanks!

**Comment by humblec** [comment]
@saad-ali Can any one of you add the milestone tag here ?

**Comment by humblec** [comment]
Looks like @saad-ali is busy as always :).. @jsafrane do you have permission to add release milestone ? If yes, can you help ?

**Comment by humblec** [comment]
@saad-ali Can you please add release milestone here ? :)

**Comment by humblec** [comment]
Thanks @idealhack !

**Comment by idealhack** [comment]
@humblec Could you provide more details why this should be in 1.12.0, or is it OK to be in 1.12.1? As we’re already delayed in this cycle, we may not able to include those PRs which is not critical urgent. So I hope there’re reasons for this :)

**Comment by idealhack** [comment]
We’ll leave the milestone label here, and this should be cherry-picked to a future patch release of 1.12

**Comment by humblec** [comment]
@idealhack its really critical and in absense of this patch, the volume mount failure reason is completely unknown to user/admin and there is no way to identify the root cause and fix, which is critical in nature. I would like to have this in `1.12` and its a regression.

**Comment by idealhack** [comment]
@humblec Thanks!

**Pull request merged**